src/components/Button.js

```diff
@@ -2,18 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { btnName } = props;
+  const { btnName, color, wide } = props;
+  let size = wide;
+  if (wide) {
+    size = '50%';
+  } else size = '25%';
+
   return (
-    <button type="button">{btnName}</button>
+    <button
+      type="button"
+      style={{ backgroundColor: color, width: size }}
+    >
+      {btnName}
+    </button>
   );
 };
 
 Button.propTypes = {
   btnName: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
 };
 
 Button.defaultProps = {
   btnName: '',
```

Comment:
Please, let's delete this. `btnName` should not have a default value. See next comment.